src/mpl/include/mpl_thread_posix.h

```diff
@@ -43,7 +43,10 @@ void MPL_thread_create(MPL_thread_func_t func, void *data, MPL_thread_id_t * id,
         *(same_) = pthread_equal(*(id1_), *(id2_)) ? TRUE : FALSE;	\
     } while (0)
 
-#define MPL_thread_yield MPL_sched_yield
+#define MPL_thread_yield(mutex_ptr_) \
```

Comment:
The changes in this commit should be completely dropped. See my comments in the earlier commits